site/content/contribute/getting-started/code-reviews.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Code Reviews"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+All changes to the product must be reviewed.
+
+* User experience changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
+* Code changes must be reviewed by at least two [core committers](http://localhost:1313/contribute/getting-started/core-committers/#core-committers).
+* Documentation changes should be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers) and at least one [core committer](/contribute/getting-started/core-committers/#core-committers).
+
+If you are not a core committer
+-------------------------------
+
+1. Submit your pull request.
```

Comment:
May be worth referencing this checklist: https://developers.mattermost.com/contribute/getting-started/contribution-checklist/